### ISSUE TITLE
Delegation with different domain / Domains not selectable in SOGo although all necessary settings done

### ISSUE THREAD

**Issue by pbaeumel** [mailcow/mailcow-dockerized#1426]
Dear Andre ( @andryyy),

thanks for this great piece of mailserver you are offering here.

Unfortunately I have some problems with delegation of mailbox in a multiple-domain setup.
I carefully followed the documentation, googled around and think that i made all necessary settings but it didn`t work out so far; so I open this issue.

Given details:
3 Domains: bxxxxxxl.com, zxxxxxxm.com und gxxxxl.by

Setup in web-interface of mailbox pxxxxxx@bxxxxxl.com:
![2018-05-25_220221_anon](https://user-images.githubusercontent.com/6563984/40572432-a1d287a8-60ac-11e8-8552-9090bf584d52.jpg)

Related content of sogo.conf (Note: This should be included as step in documentation! Or even better settable from the GUI ;-)  ):
`SOGoDomainsVisibility = ((bxxxxxl.com, gxxxxl.by, zxxxxxxx.com));`

I then restarted SOGo.

Result in mailbox bxxxx@gxxxxl.by:
![2018-05-26_063339_anon](https://user-images.githubusercontent.com/6563984/40572521-bf4ddad8-60ae-11e8-8da1-5a18369c85cd.jpg)
 **_No user from different domain is shown!?_**

Any idea why it doesn`t work? Have I missed a step?

Best regards,
MacGyver

**Comment by andryyy** [comment]
Hi,

    Related content of sogo.conf (Note: This should be included as step in documentation! Or even better settable from the GUI ;-) ):

No, it is documented very well in SOGo. But yes, it might be configurable from within the UI someday.

![grafik](https://user-images.githubusercontent.com/2972950/40573994-cfaa1358-60ca-11e8-9451-39b73702f937.png)

Not a problem here.

Do you see two GAL in the address book section of SOGo?

**Comment by pbaeumel** [comment]
Hi,

unfortunately only one GAL (User of same domain) is visable:
![2018-05-26_101958_anon](https://user-images.githubusercontent.com/6563984/40574193-91e27fc0-60ce-11e8-93ca-04ec4303dafd.jpg)

For me it seems that the sogo.conf setting does not work; but why?

Best regards,
MacGyver

**Comment by andryyy** [comment]
Can you please post your whole sogo.conf and also run `docker-compose restart sogo-mailcow` in case it didn't restart SOGo properly.

**Comment by pbaeumel** [comment]
I have had not used the Web-GUI for SOGo-Restart previously.
I have had issued the following commands
`docker-compose stop
docker-compose up -d`

sogo.conf:
```
{
    SOGoCalendarDefaultRoles = (
        PublicViewer,
        ConfidentialDAndTViewer,
        PrivateDAndTViewer
    );

    WOWorkersCount = "20";
    SOGoACLsSendEMailNotifications = YES;
    SOGoAppointmentSendEMailNotifications = YES;
    SOGoDraftsFolderName = "Drafts";
    SOGoJunkFolderName= "Junk";
    SOGoMailDomain = "sogo.local";
    SOGoEnableEMailAlarms = NO;
    SOGoFoldersSendEMailNotifications = YES;
    SOGoForwardEnabled = YES;
    SOGoUIAdditionalJSFiles = (js/theme-blue.js);

    // Multi-domain setup
    // Domains are isolated, you can define visibility options here.
    // Example:

    SOGoDomainsVisibility = ((bxxxxxl.com, gxxxxl.by, zxxxxxxm.com));

    SOGoIMAPServer = "imap://dovecot:143/?tls=YES";
    SOGoSieveServer = "sieve://dovecot:4190/?tls=YES";
    SOGoSMTPServer = "postfix:588";
    WOPort = "0.0.0.0:20000";
    SOGoMemcachedHost = "memcached";

    SOGoLanguage = English;
    SOGoMailAuxiliaryUserAccountsEnabled = YES;
    SOGoMailCustomFromEnabled = YES;
    SOGoMailingMechanism = smtp;
    SOGoSMTPAuthenticationType = plain;

    SxVMemLimit = 384;

    SOGoMaximumPingInterval = 354;

    SOGoInternalSyncInterval = 30;
    SOGoMaximumSyncInterval = 354;

    // 100 seems to break some Android clients
    //SOGoMaximumSyncWindowSize = 100;
    // This should do the trick for Outlook 2016
    SOGoMaximumSyncResponseSize = 2048;

    WOWatchDogRequestTimeout = 10;
    WOListenQueueSize = 300;
    WONoDetach = YES;

    SOGoIMAPAclConformsToIMAPExt = Yes;
    SOGoPageTitle = "SOGo Groupware";
    SOGoFirstDayOfWeek = "1";

    SOGoSieveFolderEncoding = "UTF-8";
    SOGoPasswordChangeEnabled = NO;
    SOGoSentFolderName = "Sent";
    SOGoMailShowSubscribedFoldersOnly = NO;
    NGImap4ConnectionStringSeparator = "/";
    SOGoSieveScriptsEnabled = YES;
    SOGoTrashFolderName = "Trash";
    SOGoVacationEnabled = YES;

    SOGoCacheCleanupInterval = 900;
    SOGoMaximumFailedLoginCount = 10;
    SOGoMaximumFailedLoginInterval = 900;
    SOGoFailedLoginBlockInterval = 900;

    MySQL4Encoding = "utf8mb4";
  //SOGoDebugRequests = YES;
  //SoDebugBaseURL = YES;
  //ImapDebugEnabled = YES;
  //SOGoEASDebugEnabled = YES;
  //LDAPDebugEnabled = YES;
  //PGDebugEnabled = YES;
  //MySQL4DebugEnabled = YES;
  //SOGoUIxDebugEnabled = YES;
  //WODontZipResponse = YES;
    WOLogFile = "/dev/sogo_log";
}

```

**Comment by andryyy** [comment]
Do not run stop. Run down and up -d to start clean.  Stop and start does also not care about the sequence.

Do you use a Umlautdomain and typed it with the Umlaut in sogo.conf?

You see there is something wrong in your sogo.conf, it is not a general bug in mailcow. I don’t have time to check this remote, sorry. But I’m sure it is just a small typo in the syntax.

**Comment by pbaeumel** [comment]
No I didn`t use Umlautdomains in sogo.conf.
Could I contact you for commercial support in this matter?

Best regards,
MacGyver

**Comment by andryyy** [comment]
Huh, strange. I have a few minutes to spare now, are you there? :-) Mail me to info@servercow.de

**Comment by pbaeumel** [comment]
Dear @andryyy,

thanks for your offer.
I have found the problem myself: Problem exists between chair and keyboard ;-)

I have had two vServers rented; one which is already cancelled and one which will remain.
For the first tryout I have installed Mailcow on the already cancelled server and then just copied over an image to the remaining server.

What I didn´t change (and seen up until 2 days now) was my SSH-program which was set to the "old" server.
So I always changed the sogo.conf file on the wrong server... *tilt*

By changing the file on the right server; everything works as expected now.

Sorry for opening that useless issue.

Best regards,
MacGyver
